topics/02_Controls/igCombo/07_igCombo_Keyboard_Navigation.md

```diff
@@ -33,15 +33,24 @@ Press| To| When
 <kbd>Alt</kbd> + <kbd>Up arrow</kbd>|Close drop down| Drop down is opened
 <kbd>Ctrl</kbd> + <kbd>Home</kbd>|Move to very first item in drop down| Drop down is opened
 <kbd>Ctrl</kbd> + <kbd>End</kbd>|Move to very last item| Drop down is opened
+<kbd>Page Up</kbd>|Move to and selects the first visible item| Drop down is opened and multiselection is enabled
+<kbd>Page Down</kbd>|Move to and selects the last visible item| Drop down is opened and multiselection is enabled
+<kbd>Up arrow</kbd>|Move to and selects the prev item; Close drop down if opened | Drop down is opened or closed
+<kbd>Down arrow</kbd>|Move to and selects next item; Open drop down if closed| Drop down is opened or closed
+<kbd>Esc key</kbd>|Clear content when there isn’t selected item | There is content in the input
+<kbd>Esc key</kbd>|Close the drop down| The drop down is opened
+<kbd>Enter key</kbd>|Close the drop down| There is selected item
+
+##<a id="multiselection"></a>Multiselection
+
+Press| To| When
+---|---|---
 <kbd>Page Up</kbd>|Move to first visible item| Drop down is opened
 <kbd>Page Down</kbd>|Move to last visible item| Drop down is opened
 <kbd>Up arrow</kbd>|Move to prev item; Close drop down if opened | Drop down is opened or closed
-<kbd>Down arrow</kbd>|Move to next item; Open drop down if closed| Drop down is opened or closed
-<kbd>Esc key</kbd>|Clear content when there isn’t selected item | There is content in the input
-<kbd>Esc key</kbd>|Close the drop down| The drop down is opened
+<kbd>Down arrow</kbd>|Move next item; Open drop down if closed| Drop down is opened or closed
 <kbd>Enter key</kbd>|Select the active item| There is active item
 
-##<a id="multiselection"></a>Multiselection
```

Comment:
Again, why is the multi selection anchor removed?

Reply:
![untitled](https://user-images.githubusercontent.com/14248932/36089947-0ccf8108-0fe7-11e8-9803-b88608866cda.png)